homeassistant/components/recorder/__init__.py

```diff
@@ -646,6 +653,8 @@ def _migrate_schema_and_setup_run(self, current_version) -> bool:
     def _run_purge(self, keep_days, repack, apply_filter):
         """Purge the database."""
         if purge.purge_old_data(self, keep_days, repack, apply_filter):
+            # If we have auto purge run
```

Comment:
Need to make this comment clear that we run the cleanups at the end of a purge or auto purge